### DISCUSSION TITLE
Is there any limitation when saving cartridge to .png?

### DISCUSSION THREAD

**Discussion by herbix** [nesbox/TIC-80#2307] [Q&A]
I'm trying to save a .tic cartridge to .png, which contains ~373KB code and 8 banks of tiles and sprites. tic80.exe crashes when I run `save cart.png`. Does it hit any limitation of png format cartridge?

**Comment by nesbox** [comment]
pls provide the cart to investigate

**Reply by herbix** [reply]
You can find the cart here: https://static.chaofan.io/8bitsts/cart.tic

**Comment by bztsrc** [comment]
> Does it hit any limitation of png format cartridge?

There should be no practical limit (to be precise, up to 2 Gigabytes, which you'll never reach). Originally tic was saved in the cartridge using steganography, which is very limited (about 96k), but now also saved as a PNG chunk. If the cartrdige is bigger than the steganography's limit then only the PNG chunk method supposed to be used.

> tic80.exe crashes when I run

It shouldn't, and that's a bug if it does. I'll look into that.

**Comment by herbix** [comment]
This has been converted to issue #2311 .
